conf.go

```diff
@@ -32,6 +32,46 @@ type Sourcer interface {
 	Source(fld Field) (string, bool)
 }
 
+const versionKey = `SVN`
+const descKey = `Desc`
+
+// Version provides the abitily to add version and description
+type Version struct {
+	SVN  string // String Version
+	Desc string
+}
+
+// Details provides output to display the application version and description on the command line.
+func Details(namespace string, v interface{}) (string, error) {
+	fields, err := extractFields(nil, v)
+	if err != nil {
+		return "", err
+	}
+
+	var str strings.Builder
+
+	for i := range fields {
+		if fields[i].Name == versionKey && fields[i].Field.Len() > 0 {
```

Comment:
I hate else clauses, can we remove it. Try a continue inside the if or use a naked switch.

Reply:
Added the continue, I think it's more readable then the naked switch.